rpcs3/Emu/RSX/GL/rsx_gl_texture.cpp

```diff
@@ -136,7 +136,7 @@ namespace
 		case CELL_GCM_TEXTURE_A1R5G5B5:
 		case CELL_GCM_TEXTURE_R5G6B5:
 		case CELL_GCM_TEXTURE_A8R8G8B8:
-		case CELL_GCM_TEXTURE_R6G5B5:
+		case CELL_GCM_TEXTURE_R6G5B5: return { GL_ALPHA, GL_GREEN, GL_RED, GL_BLUE };
```

Comment:
@raven02 I'll try to see if I can get terraria to load with some workarounds. It uses the 16-wide formats extensively in the menus. Which other games affected by this change?